src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -279,12 +278,14 @@ static int add_mount(
 static int path_is_busy(const char *where) {
         int r;
 
+        assert(where);
+
         /* already a mountpoint; generators run during reload */
         r = path_is_mount_point(where, NULL, AT_SYMLINK_FOLLOW);
         if (r > 0)
                 return false;
 
-        /* the directory might not exist on a stateless system */
+        /* The directory will be created by the mount or automount unit when it is started. */
         if (r == -ENOENT)
                 return false;
 
```

Comment:
and here too, btw, looking at this, i think we should convert ENOTDIR from the dir_is_empty() call to false